version/git.go

```diff
@@ -39,7 +39,7 @@ func git(ctx context.Context, gitDir *dagger.Directory, dir *dagger.Directory) (
 	if err != nil {
 		return nil, err
 	}
-	if slices.Contains(entries, ".git") {
+	if slices.Contains(entries, ".git/") || slices.Contains(entries, ".git") {
```

Comment:
When does `.git` not have a `/`?

Reply:
Ah, because this module needs to work on both a dev version of dagger, and a stable version. Because there's version compat, it'll technically keep working on the dev build without the slash until we upgrade, but I wanted to add this in now, so I don't forget it later.

I've added a comment though, and will remove it during the next improve-releasing PR.